backend-tests/tests/test_deployments.py

```diff
@@ -571,6 +571,14 @@ def setup_devices_and_management_st(nr_devices=100):
     devs = make_accepted_devices(devauthd, devauthm, utoken, "", nr_devices)
     # wait for devices to be provisioned
     time.sleep(3)
+    if len(deploy_to_group) > 0:
+        for device in devs[:-1]:
```

Comment:
Question:
I'm not into this, but why strip off one device?

Reply:
thats part of the test logic. one of the tests check that last device was not part of the deployment.

Reply:
Makes sense :100: